coordinator/src/services/translation/dispense/release.ts

```diff
@@ -0,0 +1,60 @@
+import * as hl7v3 from "../../../models/hl7-v3"
```

Comment:
seems like based on the new folder structure this file would be `translation/request/dispense/release.ts`